pkg/plugins/golang/v3/scaffolds/internal/templates/dockerfile.go

```diff
@@ -41,6 +41,11 @@ func (f *Dockerfile) SetTemplateDefaults() error {
 const dockerfileTemplate = `# Build the manager binary
 FROM golang:1.18 as builder
 
+# TARGET_GOOS and TARGET_GOARCH are passed as build argument
+# set by default respectivelly to linux and amd64
+ARG TARGET_GOOS=linux
+ARG TARGET_GOARCH=amd64
```

Comment:
/hold

based on : https://github.com/kubernetes-sigs/kubebuilder/issues/2850

We need to see what is the best way / approach to help those that want to build images for multi-archs